watertap/examples/flowsheets/electrodialysis/electrodialysis_1stack_conc_recirc_ui.py

```diff
@@ -0,0 +1,634 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2024, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+from watertap.core.solvers import get_solver
+from watertap.ui.fsapi import FlowsheetInterface
+from watertap.examples.flowsheets.electrodialysis.electrodialysis_1stack_conc_recirc import (
+    build,
+    _condition_base,
+    initialize_system,
+    solve,
+)
+from pyomo.environ import units as pyunits
+
+
+def export_to_ui():
+    return FlowsheetInterface(
+        name="Electrodialysis with concentrate recirculation",
+        do_export=export_variables,
+        do_build=build_flowsheet,
+        do_solve=solve_flowsheet,
+    )
+
+
+def export_variables(flowsheet=None, exports=None, build_options=None, **kwargs):
+    fs = flowsheet
+    # --- Input data ---
+    # Feed conditions
+    exports.add(
+        obj=fs.feed.properties[0].flow_vol_phase["Liq"],
+        name="Feed solution volume flowrate",
+        ui_units=pyunits.m**3 / pyunits.s,
+        display_units="m^3 s^-1",
```

Comment:
For ease of readability and to be consistent with other GUIs, we should display fractions and exponents like so, where possible
```suggestion
        display_units="m3/s",
```

Reply:
I prefer writing derived units all in products to avoid confusions in dimensional analysis.  There are quantities carrying more than 2 base units.